junit5/src/main/java/org/jboss/weld/junit5/auto/AddClasspaths.java

```diff
@@ -0,0 +1,21 @@
+package org.jboss.weld.junit5.auto;
+
+
+import java.lang.annotation.*;
+
+
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.TYPE)
+@Repeatable(AddClasspaths.All.class)
+public @interface AddClasspaths {
```

Comment:
Pls remove this functionality, we're not going to introduce another classpath magic.

Reply:
This seems to be shortsighted. This functionality is *_very_* useful when adding bean classes from an external library. The only alternative is to use `AddPackages` and interrogate the library to ensure you list all the packages from the library; which may change with each release. For example when using the DeltaSpike library we had to list about 10 different packages (possibly some private) to use the library and this annotation replaces that with a reference to a single public class from DeltaSpike.

I would urge we leave this in. As I have stated this implementation comes from a large library of real world unit tests and it is very useful. We could discuss what other "classpath magic" discovery methods are possible.